patches/chrome-installer-mini_installer-BUILD.gn.patch

```diff
@@ -35,3 +35,16 @@ index 3f65aa27ca38e9721493b4e3fe7a40593ab75e0b..e5a8172dbafdd3982e4ed50de1770fbd
      ]
  
      if (enable_hidpi) {
+@@ -270,7 +273,11 @@ template("generate_mini_installer") {
+ 
+ generate_mini_installer("mini_installer") {
```

Comment:
Do we need to make this change? I don't think we care if the filename is chrome.dll vs brave.dll. If that's the only reason we are changing it then I think we should leave it alone, or is there something else?

Reply:
Yep, it would be happen if we try to build all targets. I just tested only brave target.
I'll revert to use `chrome.dll`.